content/pt-br/docs/reference/issues-security/security.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Informações de Segurança e Divulgação do Kubernetes
+aliases: [/security/]
```

Comment:
Hi there, @italofernandes13!

I just noticed this alias must be updated to `/pt-br/security/` (sorry about that!! 😅). Would you mind updating this? It's a required change to merge this PR.

Thanks in advance!